kata-deploy/action/test-kata.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+function die() {
+	msg="$*"
+	echo "ERROR: $msg" >&2
+	exit 1
+}
+
+function waitForProcess() {
+        wait_time="$1"
+        sleep_time="$2"
+        cmd="$3"
+        while [ "$wait_time" -gt 0 ]; do
+                if eval "$cmd"; then
+                        return 0
+                else
+						echo "waiting"
```

Comment:
I'm guessing this could be

                                                             tab

issues? :smile:

Reply:
let me wrangle vim.